specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/CommonDefinitions.json

```diff
@@ -1955,6 +1955,14 @@
           "description": "<code>true</code> if HTTP logging is enabled; otherwise, <code>false</code>.",
           "type": "boolean"
         },
+        "acrUseManagedIdentityCreds": {
+          "description": "Flag to use Managed Identity Creds for ACR pull",
+          "type": "boolean"
+        },
+        "acrUserManagedIdentityID": {
```

Comment:
Would you do a quick check whether service uses `ID` or `Id`?

Rest is good for me.